Backpack/BarChart/Classes/BarChartBar.swift

```diff
@@ -20,10 +20,10 @@ import UIKit
 
 @objcMembers
 @objc(BPKBarChartBar)
-public final class BPKBarChartBar: UIControl {
+public final class BPKBarChartBar: UIView {
```

Comment:
Make the bar a simple UIView, we only used UIControl to facilitate isSelected as a property

Reply:
Any cons in leaving this a UIControl? Given is an intractable element and UIControl inherits from UIView, seems alright to leave it like that?

Reply:
The problem is that we then need to set userInteractionEnabled = false on it, to allow selection of the cell. 
There was no point in having it as a UIControl, if we do and disable interaction it will announce as disabled :/